src/main/scala/util/CoreMonitor.scala

```diff
@@ -3,17 +3,25 @@
 
 package freechips.rocketchip.util
 
-import chisel3._
+import Chisel._
+
+// this class is used to expose some internal core signals
+// to verification monitors
+case class CoreSignalMonitor(
+  val clock: Clock,
+  val reset: Reset,
+  val xLen: Int,
+  val cease: Bool,
+  val reg_mscratch: UInt,
+  val hartid: UInt)
 
-// this bundle is used to expose some internal core signals
-// to verification monitors which sample instruction commits
 class CoreMonitorBundle(val xLen: Int) extends Bundle with Clocked {
```

Comment:
So do you actually still need this Bundle?

Reply:
You're correct, this is no longer needed.